app/backend/src/clj/teet/contract/contract_db.clj

```diff
@@ -234,7 +234,15 @@
     {:target target
      :activity {:activity/manager (user-model/user-name (:activity/manager activity))
                 :activity/name (:activity/name activity)
-                :activity/tasks (:activity/tasks activity)}
+                :activity/tasks (reduce (fn [acc task]
+                                          (conj acc (merge task
+                                                           {:navigation-info
+                                                            {:page :activity-task
+                                                             :params {:project project-id
+                                                                      :activity activity-id
+                                                                      :task (str (:db/id task))}}}
+                                                           {:owner (user-model/user-name (:activity/manager activity))})))
+                                        [] (:activity/tasks activity))}
```

Comment:
Could be written more simply with `mapv` instead of `reduce`:
```
:activity/tasks (mapv (fn [task] (merge task <...>)) (:activity/tasks activity))
```